include/swift/Syntax/TokenKinds.def

```diff
@@ -23,6 +23,7 @@
 ///     SIL_KEYWORD(kw)
 ///   POUND_KEYWORD(kw)
 ///     POUND_OBJECT_LITERAL(kw, desc, proto)
+///     POUND_OLD_OBJECT_LITERAL(kw, new_kw, old_arg, new_arg)
```

Comment:
This code was removed in https://github.com/apple/swift/commit/bdcd87754d181fcb26b23276b8a82129e027f8b9. Same for the other changes in this file.

Reply:
This got in accidentally during rebasing.